src/test/addresses_mainnet.sol

```diff
@@ -360,6 +360,30 @@ contract Addresses {
         addr["RWA009_A_URN"]                    = 0x1818EE501cd28e01E058E7C283E178E9e04a1e79;
         addr["RWA009_A_JAR"]                    = 0x6C6d4Be2223B5d202263515351034861dD9aFdb6;
         addr["RWA009_A_OUTPUT_CONDUIT"]         = 0x508D982e13263Fc8e1b5A4E6bf59b335202e36b4;
+        addr["RWA010"]                          = 0x20C72C1fdd589C4Aaa8d9fF56a43F3B17BA129f8;
+        addr["PIP_RWA010"]                      = 0xfBAa6a09A39D485a5Be9F5ebfe09C602E63b21EF;
```

Comment:
This should be removed along with the 11,12,13 versions.

Reply:
done

Reply:
Nevermind.